README.md

```diff
@@ -66,12 +66,14 @@ In order to do a manual installation of the AEP SDK `.aar` binaries, complete th
 
 __Important__ - _all_ AEP SDK Android binaries depend on `core-1.x.x.aar`.
 
-## Installing the AEP SDK for iOS
+## Installing the AEP SDK (2.x) for iOS
 
-> __Important__ 
+> ⚠️ __Important__ 
 > 
-> Newer versions of the AEP SDK for iOS are distributed using XCFrameworks. If using the XCFramework distribution, Xcode 12 or newer is required and Cocoapods 1.10.0 or newer is required (where applicable).
+> A newer version of the [AEP Mobile SDKs (3.x)](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift) is now available! These SDKs are open source, bring many exciting new features and include support for Swift Package Manager (SPM), CocoaPods and XCFrameworks.
+> If you are currently using our Objective-C (ACP-prefix 2.x) libraries, please see the [Migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift) guide for next steps.
 
+__Important__ - Newer versions of the AEP SDK for iOS are distributed using XCFrameworks. If using the XCFramework distribution, Xcode 12 or newer is required and Cocoapods 1.10.0 or newer is required (where applicable).
```

Comment:
nit: Use "latest" instead of "newer" here, "The latest versions of the AEP SDK ..."